canonical-web-and-design/charmhub.io
# Remove unused templates and use image template for more images

## Done

- Remove unused templates and JS
- Use image template for more images
- Remove (required) from the contact-us form

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://localhost:8045
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Go to all the pages and see that everything looks and works as expected


## Issue / Card

Fixes #424 

## Screenshots

[if relevant, include a screenshot]
